### DISCUSSION TITLE
Does ImageMagick support .dcp camera profiles?

### DISCUSSION THREAD

**Discussion by daviddellarossa** [ImageMagick/ImageMagick#2775] [Help] [answered]
Hello,

I managed to successfully apply .icc profiles, but I haven't found a way of applying .dcp camera profiles.
Does ImageMagick support .dcp camera profiles?

Thanks

David

**Comment by snibgo** [comment, marked as answer]
As far as I know, IM can't do anything with DCP profiles.

IM uses LittleCMS for colour management, and LittleCMS uses only ICC profiles.
